frontend/packages/ceph-storage-plugin/src/components/block-pool/dashbaord/details-card.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
```

Comment:
Move Dashboards to `ceph-storage-plugin/src/components/dashboards/block-pool`

Reply:
ack